.github/workflows/build-hyrise.yml

```diff
@@ -0,0 +1,43 @@
+# How to test this:
+#   In the root of the repository, run:
+#     nix-shell -p act --run "act"
+#   Act is a software that allows you to simulate GitHub Actions CIs locally. It works best in container environments that are not self-hosted.
```

Comment:
Works best? Doesn't act require Docker? What would be the act non-self-hosted variant?

Reply:
You are right, it is required. I just wanted to say that self-hosted environments are hard to simulate, bc people often install stuff on their machines that is not mentioned in the CI build file :/ Had this experience quite a lot in recent times.